sim/host.go

```diff
@@ -18,24 +18,29 @@ type simHost struct {
 	gpbft.Verifier
 	gpbft.Clock
 
-	id  gpbft.ActorID
-	sim *Simulation
+	id     gpbft.ActorID
+	sim    *Simulation
+	pubkey gpbft.PubKey
```

Comment:
@Kubuxu re my comments on #188 , putting the pubkey here seems reasonable and may avoid the power table plumbing.

Reply:
The power table plumbing is external to the sim, as F3 won't know for which participants it will be signing. Only the application itself knows that.